enterprise/server/cmd/executor/executor.go

```diff
@@ -114,6 +115,10 @@ func GetConfiguredEnvironmentOrDie(configurator *config.Configurator, healthChec
 		log.Infof("No authentication will be configured: %s", err)
 	}
 
+	xcodeLocator := xcode.NewXcodeLocator()
+	xcodeLocator.Locate()
```

Comment:
I think you could have "NewXcodeLocator()" call locate() internally before returning, then remove the "Locate()" method from the interface, and from being a public method

Reply:
Done. Also added support for SDKs, dropped enableXcodeOverride, and simplified the CFString stuff (I think all of that craziness was only needed when allocating a new string - which we don't do).